Pcap++/src/PcapLiveDeviceList.cpp

```diff
@@ -330,12 +370,13 @@ PcapLiveDevice* PcapLiveDeviceList::getPcapLiveDeviceByName(const std::string& n
 
 PcapLiveDevice* PcapLiveDeviceList::getPcapLiveDeviceByIpOrName(const std::string& ipOrName) const
 {
-	IPAddress interfaceIP(ipOrName);
-	if (interfaceIP.isValid())
+	IPAddress interfaceIP;
```

Comment:
nit: I think we can define this variable inside the `try {..}` statement

Reply:
true, fixed.